test/E2ETests/TestAssets/TimeoutTestProjectNetCore/TimeoutTestProjectNetCore.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TestFxRoot Condition="$(TestFxRoot) == ''">..\..\..\..\</TestFxRoot>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <FrameworkIdentifier>NetCore</FrameworkIdentifier>
+    <IsPackable>false</IsPackable>
+    <!--<AppendTargetFrameworkToOutputPath>true</AppendTargetFrameworkToOutputPath>-->
```

Comment:
If we make this as false, then we will not need to specify 'netcoreapp1.1' in TimeoutTestAssemblyNetCore path here:https://github.com/Microsoft/testfx/blob/37b1ebf822786771fbcf641cfa84a39f3c83204b/test/E2ETests/Smoke.E2E.Tests/TimeoutTests.cs#L14

Reply:
Thought about it. But it is better to have a separate folder per test assembly.